easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart

```diff
@@ -39,30 +54,26 @@ class NftGridViewItem extends StatelessWidget {
             key: const Key(kGridViewTileNFTKey),
             onTap: () {
               if (context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft) {
-                final DraftsBottomSheet draftsBottomSheet = DraftsBottomSheet(
-                  buildContext: context,
-                  nft: nft,
-                );
-                draftsBottomSheet.show();
+                context.read<CreatorHubViewModel>().startPublishingFlowAgain(startPublishingFlowAgainPressed: startPublishingFlowAgainPressed);
```

Comment:
Why not this 
```
 if (context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft) {
                startPublishingFlowAgainPressed();
                return;
              }
```

Reply:
We need to write test case for this that's why need to implement this function in viewModel. Modified `startPublishingFlowAgain` function